Multisig/UI/App/UIKit/CollectiblesViewController/CollectibleTableViewCell.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  CollectibleTableViewCell.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 02.11.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class CollectibleTableViewCell: UITableViewCell {
+    @IBOutlet weak var cellWrapperView: UIView!
+    @IBOutlet weak var cellImageView: UIImageView!
+    @IBOutlet weak var cellNameLabel: UILabel!
+    @IBOutlet weak var cellDescriptionLabel: UILabel!
+
+    let cornerRadius: CGFloat = 8
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        backgroundColor = .gnoSystemWhite
+
+        cellWrapperView.backgroundColor = .gnoSystemWhite
+        cellWrapperView.layer.cornerRadius = cornerRadius
+        cellWrapperView.layer.shadowColor = UIColor.gnoShadow.cgColor
+        cellWrapperView.layer.shadowOpacity = 0.6
+        cellWrapperView.layer.shadowOffset = .init(width: 1, height: 2)
+        cellWrapperView.layer.shadowRadius = 6
+
+        cellNameLabel.setStyle(.headline)
+        cellDescriptionLabel.setStyle(.body)
+        cellDescriptionLabel.numberOfLines = 3
+    }
+
+    func configure(collectible: CollectibleViewModel) {
```

Comment:
The problem here is that view depends on this model. I would like to have 3 different methods that provide API to this class's users and that therefore will make this view potentially reusable.

```
setName(_ value: String)
setDescription(_ value: String)
setImage(with URL: URL?, placeholder: UIImage)
```

Reply:
ok

Reply:
done